paddle/fluid/inference/tests/api/analyzer_ner_tester.cc

```diff
@@ -103,6 +103,9 @@ void SetConfig(contrib::AnalysisConfig *cfg) {
   cfg->device = 0;
   cfg->specify_input_name = true;
   cfg->enable_ir_optim = true;
+#ifdef PADDLE_WITH_MKLDNN
+  cfg->_use_mkldnn = true;
+#endif
```

Comment:
please do not set `cfg->_use_mkldnn = true` here. If you want to compare MKLDNN result. please follow `analyzer_resnet50_test` to add new unit-test: `compare_mkldnn`
https://github.com/PaddlePaddle/Paddle/blob/aa87a989ecb51ddfd022dd935d0ba6fa05ea15ff/paddle/fluid/inference/tests/api/analyzer_resnet50_tester.cc#L102-L104